### DISCUSSION TITLE
Does the gtag example still work in NextJS 12?

### DISCUSSION THREAD

**Discussion by coeuscrypto** [vercel/next.js#37812] [Help]
Google reports the code isn't present, but it's added to my _app.js and _document.js etc as in the example with-google-ta-manager, and because I am using AWS Elastic Beanstalk, I have deployed the ENV Variables in the deploy script...

I am using the latest version of NextJS.

Unsure if anyone else has any experience with this, and could point me in the right direction?

**Comment by icyJoseph** [comment]
Was this the example you used? https://github.com/vercel/next.js/tree/canary/examples/with-google-tag-manager - There's no `with-google-tags` example, in https://nextjs.org/examples

**Reply by coeuscrypto** [reply]
Yes that's the example I used

**Reply by icyJoseph** [reply]
Alright, odd... when you go to your site, can you find the GTM script? And does it have the correct GTM ID? And do you see the `noscript` tags as well? If you have a link to the site I could also help out a bit faster.

**Reply by coeuscrypto** [reply]
I've ended up just going with the with-google-analytics approach, but the GTM approach I couldn't get to work. Maybe its just me

**Reply by icyJoseph** [reply]
Nah, there's probably something fishy going on.

Which part was not working? There are two moving parts the noscript for JS disabled environments and the script for the others. Was it both not working?

Well, glad to hear it's up and running now!

**Comment by coeuscrypto** [comment]
Using the GTM implementation I just couldn't get the debugger to connect to it, and I couldn't get the site to show as being a GTAG property, it was definitely passing in the GTM ID, and I could see the network response returning the source code from googletagmanager.com.

It could have been an issue at the GTM side rather than NextJS, hence why I was trying to narrow down others experiences!
